src/jquery.uls.core.js

```diff
@@ -189,6 +189,10 @@
 			}
 
 			this.visible();
+
+			// Hide the selector panel when clicking outside of it
+			hider = $.proxy( this.cancel, this );
+			setTimeout( function() { $( 'html' ).one( 'click', hider ); } );
```

Comment:
This pattern will work, I am not sure whether  the approach in https://github.com/wikimedia/jquery.uls/pull/202 has any cons or pros compared to this.